src/osu_p2p.jl

```diff
@@ -20,15 +22,19 @@ function run_osu_p2p(benchmark::MPIBenchmark, func::Function, conf::Configuratio
     func(conf.T, 1, 10, comm, conf.window_size)
 
     if iszero(rank)
-        print_header(io) = cal_bandwidth == true ? println(io, "size (bytes),iterations,bandwidth (MB/s)") :  println(io, "size (bytes),iterations,latency (seconds)")
-        print_result(io, bytes, iters, result) = println(io, bytes, ",", iters, ",", result)
-        println(conf.stdout, "----------------------------------------")
-        println(conf.stdout, "Running benchmark ", benchmark.name, " with type ", conf.T, " on ", nranks, " MPI ranks")
-        println(conf.stdout)
-        print_header(conf.stdout)
+        println(conf.stdout, "# ", benchmark.name, " with type ", conf.T, " on ", nranks, " MPI ranks")
+        if cal_bandwidth == false
+            println(conf.stdout, "size (bytes),iterations,latency (us)")
+        else
+            println(conf.stdout, "size (bytes),iterations,bandwidth (MB/s)")
```

Comment:
Nitpick: for booleans I find easier to just check the boolean itself, rather than equality to `true`/`false`
```suggestion
        if cal_bandwidth
            println(conf.stdout, "size (bytes),iterations,bandwidth (MB/s)")
        else
            println(conf.stdout, "size (bytes),iterations,latency (us)")
```